### DISCUSSION TITLE
Uniswap Labs to charge a 0.15% fee on select frontends and assets

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#6747] [General]
«After the new fee structure is in place, these assets will be subject to the charge: USDC, ETH, DAI, agEUR, EUROC, LUSD, XSGD, WETH, WBTC, USDT, and GUSD. Moreover, the input and output tokens will also be subject to the fee.»

https://www.cryptonewsz.com/uniswap-labs-to-charge-0-15-percent-fee-on-select-frontends-and-assets/

**Comment by abdrasulov** [comment]
They say "This fee will only be applicable to the Uniswap.org frontend and the Uniswap web app". In Unstoppable we do not use their frontends. We interact directly with their smart contracts in blockchains. So the fee changes should not affect Unstoppable.

**Reply by serrq** [reply]
My English is fragile. I am still learning it. Frontend is a new word for me. Thank you.

**Comment by Veneta9Anderson9** [comment]
Why is the gas jumping so much?

**Reply by Tuckerweb3** [reply]
Gas is rising because Eth and Btc are rising.
